add-ons/istio/README.adoc

```diff
@@ -0,0 +1,98 @@
+:linkattrs:
+
+= Istio Add-on
+
+This add-on deploys the https://Istio.io/[Istio] **servicemesh** on to Minishift-provisioned OpenShift cluster.
+
+NOTE: The add-on was tested with Istio release v0.2.12
+
+== Minishift profile setup
+
+The standard Istio setup deploys around four to five applications like istio-ingress, istio-egress, istio-pilot, grafana, prometheus and their related resources
+such as configmaps, secrets, https://kubernetes.io/docs/concepts/api-extension/custom-resources/[Custom Resource Definitions].
```

Comment:
Why do you have a link for one of these, but not any of the others?

Also, please use "and" before the last item in the list. :)

Reply:
done